src/Vehicle/RemoteIDManager.cc

```diff
@@ -0,0 +1,424 @@
+/****************************************************************************
+ *
+ * (c) 2009-2022 QGROUNDCONTROL PROJECT <http://www.qgroundcontrol.org>
+ *
+ * QGroundControl is licensed according to the terms in the file
+ * COPYING.md in the root of the source code directory.
+ *
+ ****************************************************************************/
+
+#include "RemoteIDManager.h"
+#include "QGCApplication.h"
+#include "SettingsManager.h"
+#include "RemoteIDSettings.h"
+#include "QGCQGeoCoordinate.h"
+#include "PositionManager.h"
+
+#include <QDebug>
+
+QGC_LOGGING_CATEGORY(RemoteIDManagerLog, "RemoteIDManagerLog")
+
+#define AREA_COUNT 1
+#define AREA_RADIUS 0
+#define SENDING_RATE_MSEC 1000
+#define ALLOWED_GPS_DELAY 5000
+#define RID_TIMEOUT 2000 // Messages should be arriving at 1 Hz, so we set a 2 second timeout
```

Comment:
Might be better to set it to 2.5, otherwise a single drop might or might not trigger a timeout
```suggestion
#define RID_TIMEOUT 2500 // Messages should be arriving at 1 Hz, so we set a 2 second timeout
```

Reply:
I agree, this should be fixed now!